app/views/admin/_general.html.erb

```diff
@@ -33,6 +33,7 @@
         <li><%= link_to 'Manage assay types', suggested_assay_types_path %></li>
         <li><%= link_to 'Manage technology types', suggested_technology_types_path %></li>
         <li><%= link_to 'Manage tags', tags_admin_path %></li>
+        <li><%= link_to 'Manage extended metadata types', administer_extended_metadata_types_path %></li>
```

Comment:
should you standardize the case of this “extended metadata” ？

Reply:
yeah, I had noticed there is an inconsistency with Workflow Class, I will capitalize them all

<img width="201" alt="image" src="https://github.com/seek4science/seek/assets/201658/8de0d181-e884-4210-98f5-5f101c65ab37">